src/XRP/xrp-client-decorator.ts

```diff
@@ -98,4 +98,17 @@ export default interface XrpClientDecorator {
    *          and the final status of the transaction.
    */
   enableDepositAuth(wallet: Wallet): Promise<TransactionResult>
+
+  /**
+   * Enables DepositPreauth and authorizes an XRPL account to send to this XRPL account.
```

Comment:
"Enables DepositPreauth" doesn't totally make sense to me... Because you "Enable deposit authorization" and then you "preauthorize a sender" ... so maybe it could just read "Authorizes an XRPL account to send to this XRPL account."

Reply:
got it, will keep that in mind - that makes way more sense